src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.jsx

```diff
@@ -25,13 +27,22 @@ const SelectTimecardPeriodType = () => {
 
   const radioName = 'timePeriod';
 
-  const { timeEntries, setTimeEntries, setSummaryErrors, setNewTimeEntry } =
-    useTimecardContext();
+  const {
+    timeEntries,
+    setTimeEntries,
+    summaryErrors,
+    setSummaryErrors,
+    setNewTimeEntry,
+  } = useTimecardContext();
 
   const handleError = (errorFields) => {
     setSummaryErrors(errorFields);
   };
 
+  useEffect(() => {
+    focusErrors(document.getElementById('summary-error-0-message'));
+  }, [summaryErrors]);
```

Comment:
I’m trying to think when we would we get an error summary as part of `SelectTimecardPeriodType` 🤔

Reply:
When the user doesn't select a type and clicks continue.